couchable/core.py

```diff
@@ -163,7 +175,7 @@ def findHandler(cls_or_name, handler_dict):
     if (cls_or_name,) in handler_dict:
         return handler_dict[(cls_or_name,)]
     elif isinstance(cls_or_name, type):
-        for type_, handler in reversed(handler_dict.items()):
+        for type_, handler in reversed(list(handler_dict.items())):
```

Comment:
`.items()` returns an iterator in Python3, to make `reversed` work this needs to be a list